content/kubermatic/master/installation/install_kkp_on_node/_index.en.md

```diff
@@ -0,0 +1,97 @@
++++
+linkTitle = "Install KKP - Quick Guide"
+title = "Get started with KKP - “KKP on single node with kubeone”"
```

Comment:
Suggestion: Get started with KKP - KKP on a single node with KubeOne on AWS

We could copy this and make guides for digitalocean, azure etc. This kind of content would help a lot.

Reply:
@shaase-ctrl  Updated the title as per Suggestion